test/steps_test.go

```diff
@@ -2616,3 +2620,32 @@ func callingAppTraceProduces(arg1 string) error {
 	}
 	return nil
 }
+
+func aSourceMapJsonFile(arg1 string) error {
+	b, err := loadResource(arg1)
+	if err != nil {
+		return err
+	}
+
+	ism := map[string]interface{}{}
+	if err := json.Unmarshal(b, &ism); err != nil {
+		return err
+	}
+
+	sourceMap, err = logic.DecodeSourceMap(ism)
+
+	return err
+}
+
+func theStringComposedOfPclineNumberEquals(arg1 string) error {
```

Comment:
@barnjamin Minor and optional:  Unless you prefer as is, can parameters like `arg1` be named according to intended use case?